### PR TITLE
[RDY] Add help dialog

### PR DESCRIPTION
**Pull request by ohuang12**
Add a short cut help dialog.
Shows when the user presses F1.

### CODE REVIEW

**Comment by Unia** [comment]
Perhaps we should not make this dialog modal, as the user might want to have it open and still interact with DNAinator to "practice". I also think that "Keyboard shortcuts" is a better term than "Hotkey bindings"; I personally have never heard of the latter.

File: dnainator-javafx/src/main/resources/fxml/dnainator.fxml [hunk 1]
```diff
@@ -65,7 +62,11 @@
           </Menu>
             <Menu mnemonicParsing="false" text="Help">
               <items>
-                <MenuItem id="about" mnemonicParsing="false" onAction="#aboutUsAction" text="About DNAinator" />
+                <MenuItem id="about" mnemonicParsing="false" onAction="#hotkeyHelpAction" text="Hotkeys">
```
Comment (Unia) [inline dnainator-javafx/src/main/resources/fxml/dnainator.fxml]: Is this `id` necessary?

**Comment by Unia** [comment]
See [this diff](https://gist.github.com/Unia/9d3c2d01a439cad34231) to drop the `ArrayList` and use varargs instead. It's a bit cleaner, imo.

**Comment by ohuang12** [comment]
Varargs indeed looks much cleaner, I will drop the `ArrayList`.

File: dnainator-javafx/src/main/resources/fxml/dnainator.fxml (hunk 1 above)
Comment (ohuang12) [inline dnainator-javafx/src/main/resources/fxml/dnainator.fxml]: No not needed, good catch.

**Comment by ohuang12** [comment]
I am not sure about "Hotkey bindings" vs "Keyboard shortcuts". If others would prefer the latter, I wouldn't mind changing it. I think both are acceptable names.

**Comment by pvanagtmaal** [comment]
Agreed on making it non-modal, and I personally prefer "Keyboard shortcuts" as well, or just "Hotkeys".

**Comment by ohuang12** [comment]
Updated it, named it "Hotkeys".

**Comment by Unia** [comment]
Can be merged by the next reviewer!

**Comment by pvanagtmaal** [comment]
Merging after travis is done :)

**Pull request merged**